pkg/setting/setting.go

```diff
@@ -854,7 +854,17 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 		return err
 	}
 	AuthProxyAutoSignUp = authProxy.Key("auto_sign_up").MustBool(true)
-	AuthProxyLDAPSyncTtl = authProxy.Key("ldap_sync_ttl").MustInt()
+
+	ldapSyncVal := authProxy.Key("ldap_sync_ttl").MustInt()
```

Comment:
you can pass in default (60) to the MustInt function. and maybe define that as a constant? AUTH_PROXY_SYNC_TTL (still needs to match the one defined in defaults.ini though.. so add that as a comment on the constant)

Reply:
was this forgotten?

Reply:
Yes, sorry!